src/mlagility/helpers/labels.py

```diff
@@ -0,0 +1,80 @@
+import os
```

Comment:
Why create a new helpers dir instead of putting this in `analysis/`?

Reply:
Although `labels.py` is currently only used within analysis it will be used by other tools that want to extract the labels from the cache. Therefore, I don't think leaving it inside "analysis" is a good choice. Any suggestions of other places this could go to? Or is it ok on helpers?

Reply:
I created a `common` dir in #54, how about that?

Reply:
That also works! Just moved things to the common dir!